zerver/context_processors.py

```diff
@@ -101,10 +101,18 @@ def zulip_default_context(request: HttpRequest) -> Dict[str, Any]:
         realm_uri = settings.ROOT_DOMAIN_URI
         realm_name = None
         realm_icon = None
+        realm_plan_type = None
+        server_admin_email = None
+
     else:
         realm_uri = realm.uri
         realm_name = realm.name
         realm_icon = get_realm_icon_url(realm)
+        realm_plan_type = realm.plan_type
+        if realm.get_human_admin_users():
+            server_admin_email = realm.get_human_admin_users().first().delivery_email
```

Comment:
Since you're calling `realm.get_human_admin_users()` twice, you could save it to a variable to clean up the code a bit and makes it easier to read. That function is also creating queries to the database, so setting it to a variable could reduce the amount of queries made.